docs/user-roles.md

```diff
@@ -0,0 +1,144 @@
+| Azure TRE developer | Extend the TRE OSS solution | |
+| Data engineer | Move data to the TRE | |
+| Information security officer | Validate and sign-off TRE deployment | |
+
+## Azure administrator
+
+Provisions the Azure TRE solution in an Azure subscription and performs tasks that require knowledge of Azure operations and has access to the Azure subscription.
+
+Example tasks:
+
+- Provision Azure TRE solution instances.
+- 2nd line support for TRE administrators, TRE workspace owners and Researchers when Azure TRE troubleshooting is required.
+- Work with the data engineer to connect the Azure TRE with the data platform.
+- Troubleshoot provisioning issues and failed deployments.
+- Manage TRE administrator users.
+- Manage data backups and restores.
+- Update the Azure TRE instances.
+- Configure log and metrics alerts.
+
+Expected skills:
+
+- Azure administration and operations.
+- Infrastructure as Code (Terraform, ARM, Git)
+- PowerShell, Bash
+
+## TRE administrator
+
+Day-to-day running and operations of the Azure TRE instance without touching Azure resources.
+
+Example tasks:
+
+- Manage workspace owner users.
+- Provision workspaces
+- Manage shared services. (e.g. Avaiable packages in Package mirror shared service.)
+- Monitor workspace usage and billing.
+- Set and manage quotas.
+
+Expected skills:
+
+- Limited or no Azure knowledge expected.
+- Python.
+
+## TRE workspace owner
+
+Owns a specific workspace and has additional privileges than the researcher within the workspace. Is most likely also a *Researcher*.
+
+Example tasks:
+
+- Manage Researcher users.
+- Export data from workspace.
+- Import data and make it available within the workspace.
+- Enable services within the workspace.
+- Monitor billing and usage of the workspace.
+
+Expected skills:
+
+- Limited or no Azure knowledge expected.
+- Python.
+
+## Researcher
+
+Has access to one specific workspace and can use all the services provisioned within that workspace.
+
+Example tasks:
+
+- Import software packages needed to conduct research (PyPi, Conda, Apt).
+- Perform research using the services in the workspace.
+
+Expected skills:
+
+- Python, R
+- Git
+- Linux
+
+## TRE service integrator
+
+Integrates workspace service types with an Azure TRE instance. This involves extending the Azure Infrastructure as Code templates to make a workspace service available within an Azure TRE instance.
+
+Example tasks:
+
+- Integrate a workspace service type with your Azure TRE instance
+
+Expected skills:
+
+- Infrastructure as Code (Terraform, ARM, Git)
+- Python, C#, PowerShell, Bash
+- Azure administration
+
+## Azure TRE developer
+
+Software developer who contributes to the development of the Azure TRE solution.
+
+Example tasks:
+
+- Modify the deployment service, management API and other components of the AzTRE solution.
+- Contribute to the Azure TRE OSS solution.
+
+Expected skills:
+
+- Git, GitHub
```

Comment:
Seems like this person should also have the same skills as the TRE service integrator - so PowerShell, Bash, Azure Administration - etc.
For consistency maybe also list the skills in the same order so it is easy to find the differences